docs/installation/install-client.rst

```diff
@@ -109,7 +109,7 @@ Whenever you need to run Flocker CLI commands, ensure you are in the virtualenv:
 .. version-prompt:: bash alice@mercury:~$ auto
 
    alice@mercury:~$ source flocker-client/bin/activate
-   alice@mercury:~$ flocker-deploy --version
```

Comment:
:question: That's been annoying me too. We should remove ``flocker-deploy`` to avoid any confusion. https://clusterhq.atlassian.net/browse/FLOC-4371